x-pack/plugins/spaces/public/views/management/edit_space/enabled_features/toggle_all_features.tsx

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { EuiContextMenuItem, EuiContextMenuPanel, EuiLink, EuiPopover } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import React, { Component } from 'react';
+interface Props {
+  onChange: (visible: boolean) => void;
+  disabled?: boolean;
+}
+
+interface State {
+  isPopoverOpen: boolean;
+}
+
+interface ToggleOption {
+  id: 'show' | 'hide';
+  text: string;
+}
+
+const options: ToggleOption[] = [
+  {
+    id: 'show',
+    text: i18n.translate('xpack.spaces.management.showAllFeaturesText', {
+      defaultMessage: 'Show all',
+    }),
+  },
+  {
+    id: 'hide',
+    text: i18n.translate('xpack.spaces.management.hideAllFeaturesText', {
+      defaultMessage: 'Hide all',
+    }),
+  },
+];
+
+export class ToggleAllFeatures extends Component<Props, State> {
+  public state = {
+    isPopoverOpen: false,
+  };
+
+  public render() {
+    const button = (
+      <EuiLink onClick={this.onButtonClick} className={'spcFeatureChangeAllLink'}>
```

Comment:
To help find this in the future we like to name classes similar to the component they are in so this should be something like : `spcToggleAllFeatures__changeAllLink`